app/src/main/res/layout/sign_up_fragment.xml

```diff
@@ -11,113 +11,117 @@
             type="org.fossasia.openevent.app.common.data.models.User" />
     </data>
 
-    <android.support.v4.widget.NestedScrollView
+    <FrameLayout
```

Comment:
OK, I see what you are saying. But why wrap this in RelativeLayout? 

1. Use CoordinatorLayout as root
2. Do not use RelativeLayout. The layout can be directly imported without needing to be wrapped with any layout